spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/ReLU.scala

```diff
@@ -30,7 +32,7 @@ import scala.reflect.ClassTag
  */
 @SerialVersionUID(1208478077576570643L)
 class ReLU[T: ClassTag](ip: Boolean = false)(
-  implicit ev: TensorNumeric[T]) extends Threshold[T](0, 0, ip) {
+  implicit ev: TensorNumeric[T]) extends Threshold[T](0, 0, ip) with DefaultOutputShape{
```

Comment:
The name is confusing. Do not use Default in the name here? It's not default, use Identity instead?

Reply:
sounds good